mget.go

```diff
@@ -25,20 +27,19 @@ type MGetRequest struct {
 	Docs []RoutableDoc
 }
 
-// NewMGetRequest instantiates an empty MGetRequest.
-// An empty MGetRequest is executable but will return zero documents because zero have been requested.
```

Comment:
Keep this extra context in the godoc comment

Reply:
done